src/pages/api/trpc/[trpc].ts

```diff
@@ -48,6 +48,7 @@ export default withAxiom(
         console.error(error);
       }
       handleTRPCError(error);
+      log.error('TRPC Error', { code: error.code, message: error.message });
```

Comment:
Not sure if we have a dev instance of axiom, so maybe just run this when in prod?

Reply:
i added a check but im not sure if the default settings already do that.